### ISSUE TITLE
NumberRange in dynamic forms

### ISSUE THREAD

**Issue by hashitha** [EdonGashi/WpfMaterialForms#19]
is NumberRange available for DynamicForms?

**Comment by mvmcjr** [comment]
I don't think so, as WPF does not officially support it, though we could implement it. You can bind it to a integer and specify its range too.

**Comment by hashitha** [comment]
Sorry, I meant to have a slider like in the old demo volume slider which uses NumberRangeSchema

![image](https://user-images.githubusercontent.com/10569142/34203923-b313dfe6-e5b7-11e7-9526-685b90e47173.png)

**Comment by mvmcjr** [comment]
@hashitha Oh alright, yes I think we missed it (Although we need @EdonGashi to confirm it). Either way, this doesn't seem difficult to implement, I'll try to add it to my fork later.

**Comment by edongashi** [comment]
A `[Slider]` attribute should be a good fit for this control. Basically we add a PropertyBuilder that checks for this attribute and returns the respective control.

For xml we can do something like 

```xml
<!-- Instead of input we do slider-->
<input type="int" ... />
<slider type="int" ... />

<!-- We also need support for a select tag -->
<select type="string" from="{Binding Items}" />
<select type="string">
  <option value="Option 1" />
  <option value="Option 2" />
  <option value="Option 3" />
</select>
```

Besides xml support new controls must also support metro and wpf. So we must consider 4 implementations for each new field type. Takes a little more time but nothing undoable.

**Comment by edongashi** [comment]
Added `[Slider(Minimum = ..., Maximum = ..., IsDiscrete = ....)]` IsDiscrete is not yet implemented.

Check Settings model for demo
